tests/Doctrine/Sniffs/Arrays/data/operatorSpacingAfterSniff.errors.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+
+$arrayWithMultipleSpacesAfterAssigmentOperator =                       [
+    0,
+    1,
+    2,
+    3,
+    4,
+    5,
+];
```

Comment:
We should probably keep all other CS rules consistent in these files, to highlight the actual issue. 

In this case, we miss strict type declaration and the EOL

Reply:
Yes, I did run `phpcs` over it, but somewhy it seems not working for me. Any idea?

```bash
root@f464869d226f:/var/www$ ./vendor/bin/phpcs tests/Doctrine/Sniffs/Arrays/data/ 


Time: 52ms; Memory: 8MB

```

Reply:
anyway, I'll fix the EOL and strict types issue